install.sh

```diff
@@ -0,0 +1,448 @@
+    install -m 755 "${TMP_DIRECTORY}/$NAME" "/usr/local/bin/hysteria"
+  fi
+}
+
+install_hysteria() {
+  # Install hysteria binary to /usr/local/bin/
+  install_file hysteria-linux-$MACHINE
+  
+  # Install hysteria configuration file to $JSON_PATH
+  # shellcheck disable=SC2153
+  if [[ -z "$JSONS_PATH" ]] && [[ ! -d "$JSON_PATH" ]]; then
+    install -d "$JSON_PATH"
+    echo -e '{\n"listen": ":36712",\n "acme": {\n"domains": [\n\n"your.domain.com"\n],\n"email": "hacker@gmail.com"\n},\n"obfs": "fuck me till the daylight",\n"up_mbps": 100,\n"down_mbps": 100\n}' | python -m json.tool > "${JSON_PATH}/config.json"
+    CONFIG_NEW='1'
+  fi
+
+  # Install hysteria configuration file to $JSONS_PATH
+  if [[ -n "$JSONS_PATH" ]] && [[ ! -d "$JSONS_PATH" ]]; then
+    install -d "$JSONS_PATH"
+    echo -e '{\n"listen": ":36712",\n "acme": {\n"domains": [\n\n"your.domain.com"\n],\n"email": "hacker@gmail.com"\n},\n"obfs": "fuck me till the daylight",\n"up_mbps": 100,\n"down_mbps": 100\n}' | python -m json.tool > "${JSONS_PATH}/${BASE}.json" 
+    CONFDIR='1'
+  fi
+}
+
+install_startup_service_file() {
+  echo "[Unit]
+Description=Hysteria, a feature-packed network utility optimized for networks of poor quality
+Documentation=https://github.com/HyNetwork/hysteria/wiki
+After=network-online.target nss-lookup.target
+Wants=network-online.target systemd-networkd-wait-online.service
+
+[Service]
+User=root
+CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
+AmbientCapabilities=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
+NoNewPrivileges=true
+ExecStart=/usr/local/bin/hysteria -c /etc/hysteria/config.json server
+Restart=on-failure
+RestartPreventExitStatus=1
+RestartSec=5
+
+[Install]
+WantedBy=multi-user.target" > /lib/systemd/system/hysteria.service
+  echo "[Unit]
+Description=Hysteria, a feature-packed network utility optimized for networks of poor quality
+Documentation=https://github.com/HyNetwork/hysteria/wiki
+After=network-online.target nss-lookup.target
+Wants=network-online.target systemd-networkd-wait-online.service
+
+[Service]
+User=root
+CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
+AmbientCapabilities=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
+NoNewPrivileges=true
+ExecStart=/usr/local/bin/hysteria -c /etc/hysteria/%i.json server
+Restart=on-failure
+RestartPreventExitStatus=1
+RestartSec=5
+
+
+[Install]
+WantedBy=multi-user.target" > /lib/systemd/system/hysteria@.service
+  echo "info: Systemd service files have been installed successfully!"
+  systemctl daemon-reload
+  SYSTEMD='1'
+}
+
+start_hysteria() {
+  if [[ -f '/lib/systemd/system/hysteria.service' ]]; then
+    if systemctl start "${HYSTERIA_CUSTOMIZE:-hysteria}"; then
+      echo 'info: Start the Hystaria service.'
+    else
+      echo '${red}error: Failed to start Hystaria service.'
+      exit 1
+    fi
+  fi
+}
+
+stop_hysteria() {
+  HYSTERIA_CUSTOMIZE="$(systemctl list-units | grep 'hysteria@' | awk -F ' ' '{print $1}')"
+  if [[ -z "$VHYSTERIA_CUSTOMIZE" ]]; then
```

Comment:
Should be `$HYSTERIA_CUSTOMIZE`

Reply:
@Mr-Sheep This is not fixed... `VHYSTERIA_CUSTOMIZE` is a typo

Reply:
sry for not noticing that typo. I've corrected it.